pkg/agent/controller/noderoute/node_route_controller_test.go

```diff
@@ -74,8 +93,7 @@ func (f *fakeIPsecCertificateManager) HasSynced() bool {
 	return true
 }
 
-func newController(t *testing.T, networkConfig *config.NetworkConfig) *fakeController {
-	clientset := fake.NewSimpleClientset()
+func newController(t *testing.T, networkConfig *config.NetworkConfig, clientset *fake.Clientset) *fakeController {
```

Comment:
Instead of adding a parameter to every test, a simpler way is to make it variadic.
```
func newController(t *testing.T, networkConfig *config.NetworkConfig, objects ...runtime.Object) *fakeController {
    clientset := fake.NewSimpleClientset(objects...)
```

Reply:
Done